webapp/src/ts/modules/contacts/contacts.component.ts

```diff
@@ -428,22 +451,166 @@ export class ContactsComponent implements OnInit, OnDestroy{
     return contact._id + contact._rev;
   }
 
-  // const setActionBarData = () => {
-  //   ctrl.setLeftActionBar({
-  //     hasResults: ctrl.hasContacts,
-  //     userFacilityId: usersHomePlace && usersHomePlace._id,
-  //     childPlaces: allowedChildPlaces,
-  //     exportFn: function() {
-  //       Export('contacts', ctrl.filters, { humanReadable: true });
-  //     },
-  //   });
-  // };
-
-  // const updateAllowedChildPlaces = () => {
-  //   XmlForms.listen('ContactsListCtrl',{ contactForms: true }, (err, forms) => {
-  //     const allowCreateLink = contactType => forms && forms.find(form => form._id === contactType.create_form);
-  //     allowedChildPlaces = childPlaces.filter(allowCreateLink);
-  //     setActionBarData();
-  //   });
-  // };
+  private setLeftActionBar() {
+    this.globalActions.setLeftActionBar({
+      hasResults: this.hasContacts,
+      userFacilityId: this.usersHomePlace && this.usersHomePlace._id,
+      childPlaces: this.allowedChildPlaces?.sort((a, b) => a.id.localeCompare(b.id)),
```

Comment:
Can we store `allowedChildPlaces` so they're already sorted? To avoid sorting them every time we update the action bar?

Reply:
It was like this in the original, but that's right, I'll move the sort to where we fetch and filter child places

Reply:
Done